ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -683,24 +771,9 @@ void Replica::send_prepare(Seqno seqno)
         break;
       }
 
-      auto& pp_root = pp->get_merkle_root();
-      if (!std::equal(
-            std::begin(pp_root),
-            std::end(pp_root),
-            std::begin(info.merkle_root)))
-      {
-        LOG_FAIL << "Merkle root between execution and the pre_prepare message "
-                    "does not match, seqno:"
-                 << seqno << std::endl;
-        break;
-      }
-
-      auto tx_ctx = pp->get_ctx();
-      if (tx_ctx != info.ctx)
+      auto merkle_info_match = compare_execution_results(info, pp);
```

Comment:
As per the previous comment, this is not the merkle info. This is the merkle root and some context data